test/expr/unary/do_expr.swift

```diff
@@ -0,0 +1,655 @@
+// RUN: %target-typecheck-verify-swift -disable-availability-checking -enable-experimental-feature ThenStatements -enable-experimental-feature DoExpressions
```

Comment:
I think you need `// REQUIRES: asserts` in here.

Reply:
Good catch!